src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -1,11 +1,25 @@
-<nav class="DevHub-Navigation {% if request.user.is_authenticated() %}DevHub-Navigation--signedin{% endif %}">
+<nav class="DevHub-Navigation {% if user_authenticated %}DevHub-Navigation--signedin{% endif %}">
   <ul>
-    <li><a href="#"><div class="Logo{% if request.user.is_authenticated() %} Logo--secondary{% endif %}" alt="{{ dev_page_title() }}" /></a></li>
-    <li><a href="#">Docs</a></li>
-    <li><a href="#">Support</a></li>
-    <li><a href="#">Blog</a></li>
-    {% if request.user.is_authenticated() %}
+    <li>
+      <a href="#" class="Logo{% if user_authenticated %} Logo--secondary{% endif %}">
```

Comment:
Super minor, but above this there's a space before the `{% if %}`, here it's after. I prefer the space outside the if, but I prefer consistency overall so do whichever you prefer as long as they're same-y 😄